apps/infra/src/core/elasticCache.ts

```diff
@@ -25,14 +25,14 @@ const SecurityGroup = new aws.ec2.SecurityGroup('LatitudeLLMCacheSg', {
 })
 
 // Create an ElastiCache cluster
-// const cacheCluster = new aws.elasticache.Cluster('LatitudeLLMCacheCluster', {
-//   engine: 'redis',
-//   nodeType: 'cache.t3.micro',
-//   numCacheNodes: 1,
-//   port: 6379,
-//   subnetGroupName: subnetGroup.name,
-//   securityGroupIds: [SecurityGroup.id],
-// })
+const cacheCluster = new aws.elasticache.Cluster('LatitudeLLMCacheCluster', {
+  engine: 'redis',
+  nodeType: 'cache.t3.micro',
+  numCacheNodes: 1,
+  port: 6379,
+  subnetGroupName: subnetGroup.name,
+  securityGroupIds: [SecurityGroup.id],
+})
```

Comment:
Why are we enabling elastic now?

Reply:
this is not elasticsearch this is redis 😅 , we use it for our cache

Reply:
https://en.wikipedia.org/wiki/Amazon_ElastiCache

Reply:
Ah lol